src/lib/account/utxo.ts

```diff
@@ -35,7 +36,10 @@ export const deriveUtxoAccountIdsAndMetadata: DeriveAccountIdsAndMetadata = asyn
           },
         ])
 
-        if (!pubkeys?.[0]?.xpub) throw new Error('failed to get public key')
+        // TODO(gomes): hdwallet snaps is borked and can't get UTXOs, this is commented so we don't crash all
+        // Unrug UTXOs and uncomment me.
```

Comment:
Revert, dis not borked anymore

Reply:
Removed TODO, but still kept the extra safety, see 2de556a40 - pretty much what the comment says WRT the switch from throwing to continuing

Also note the `typeof pubkeys?.[0]?.xpub !== 'string'` check which isn't present in develop.
This is to accommodate for MM JSON-RPC return type in case of errors: 
<img width="731" alt="image" src="https://github.com/shapeshift/web/assets/17035424/4760e3c2-a97b-4c0e-92d7-c237651342ff">